src/helpers.js

```diff
@@ -105,6 +125,11 @@ const getPrettierOptions = (editor: TextEditor) => ({
   jsxBracketSameLine: getPrettierOption('jsxBracketSameLine'),
 });
 
+const getPrettierCliOptions = (options: Object) =>
+  Object.keys(options)
+    .reduce((cliOptions, option) => [...cliOptions, `${CLI_OPTIONS[option]} ${options[option]}`], [])
+    .join(' ');
```

Comment:
Ohh 🤦‍♂️  this is what happens when I write code in the evening. This used to concatenate strings like your second suggestion, that's why `reduce`, then I changed my mind to do an array and a join and didn't notice it's now just a map.

Happy to change it to take `editor` instead of options.